src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -78,6 +82,13 @@ public function __construct(bool $magicCall = false, bool $throwExceptionOnInval
         $this->ignoreInvalidIndices = !$throwExceptionOnInvalidIndex;
         $this->cacheItemPool = $cacheItemPool instanceof NullAdapter ? null : $cacheItemPool; // Replace the NullAdapter by the null value
         $this->ignoreInvalidProperty = !$throwExceptionOnInvalidPropertyPath;
+        $this->readInfoExtractor = $this->writeInfoExtractor = new ReflectionExtractor(
+            ['set'],
+            ['get', 'is', 'has', 'can'],
```

Comment:
If the order does not matter, you can use `ReflectionExtractor::$defaultAccessorPrefixes`